packages/tools/component-generator/src/index.ts

```diff
@@ -1,159 +1,17 @@
-      "publish-component": "yarn publish --access=public"
-    },
-    "bugs": {
-      "url": "https://github.com/no-gravity-company/no-gravity-elements/issues"
-    }
-  }
-    `;
-};
-
-const getTypesTemplate = (componentName: string): string => {
-    return `export interface ${componentName}Props {
-    x: string;
-}
-    `;
-};
-
-const getStoryTemplate = (
-    componentName: string,
-    kebabCase: string,
-    componentType: string
-): string => {
-    return `import { Meta, StoryObj } from '@storybook/web-components';
-import { html } from 'lit-html';
-
-import { ${componentName}Props } from '@${componentType}/${componentName}/types';
-
-import '@no-gravity-elements/${kebabCase}';
-
-type CustomArgs = ${componentName}Props & { text: string };
-
-const meta: Meta<CustomArgs> = {
-  title: '${componentName}',
-  component: 'nge-${kebabCase}',
-  parameters: {
-    webComponents: {
-      tagName: 'nge-${kebabCase}',
-    },
-  },
-  argTypes: {
-    x: {
-      description: 'Custom prop text',
-      control: { type: 'text' },
-    },
-    text: {
-      description: 'Custom text',
-      control: { type: 'text' },
-    },
-  },
-  render: ({ x, text }) => html\` <nge-${kebabCase} x=\${x}>\${text}</nge-${kebabCase}> \`,
-};
-
-export default meta;
-type Story = StoryObj<CustomArgs>;
-
-export const Default: Story = {
-  args: {
-    text: 'Text',
-    x: 'props'
-  },
-};
-`;
-};
-
-const getUnitTestsTemplate = (
-    componentName: string,
-    componentType: string
-): string => {
-    return `import { h } from 'preact';
-import { shallow } from 'enzyme';
-
-import ${componentName} from '@${componentType}/${componentName}/${componentName}';
-
-describe('${componentName}', () => {
-  it('should match the snapshot', () => {
-    const wrapper = shallow(<${componentName} x='test'>Test</${componentName}>);
-    expect(wrapper).toMatchSnapshot();
-  });
-});`;
-};
-
-const getIntegrationTestsTemplate = (
-    componentName: string,
-    kebabCase: string
-): string => {
-    const lowerCaseName = componentName.toLowerCase();
-    return `/// <reference types="cypress" />
-
-context('${componentName}', () => {
-    beforeEach(() => {
-        cy.visit('/iframe.html?args=&id=${lowerCaseName}--default&viewMode=story');
-    });
-
-    it('should check if <nge-${kebabCase}> is present in the dom', () => {
-        cy.get('nge-${kebabCase}').should('be.visible');
-    });
-});
-    `;
-};
+import {
+    errorMessage,
+    getKebabCase,
+    getComponentTemplateWithValues,
+    getGitIgnoreTemplate,
+    getPackageJsonTemplate,
+    getTypesTemplate,
+    getStoryTemplate,
+    getUnitTestsTemplate,
+    getIntegrationTestsTemplate,
+    getSCSSTemplate,
+} from './helpers';
```

Comment:
Do we need to export these from index.ts? Feels like we could create a circular reference with this